yacron/config.py

```diff
@@ -316,6 +325,44 @@ def __init__(self, config: dict) -> None:
                     "but yacron is not running as superuser".format(self.name)
                 )
 
+    def __parse_environs(self):
+        """
+        Parse env_file and environments configuration parameters.
+
+        Opens `env_file` and loads variables, then merges them with the `environment` ones.
+        The latter will replace the former if duplicated.
+        """
+        # a temporary dictionary to prevent duplicated keys.
+        # why are you using a list?
+        environ: Dict[str, str] = {}
+
+        # if `env_file` is specified, load the file contents
+        if self.env_file is not None:
+            with open(self.env_file, 'r') as env_file:
+                file_environs = env_file.readlines()
+
+            # parse each line by stripping whitespaces and trailing newlines
+            # ignores empty lines and comments
+            # raises ConfigError if a key-value pair is not present
+            # you may want to use the `dotenv` library to do the job
+            for line in file_environs:
+                line = line.strip(' ').rstrip('\n')
+                if line.startswith('#') or not line:
+                    continue
+                if line.find('=') == -1:
```

Comment:
`line.find('=') == -1` is not very idiomatic Python.  Better would be `'=' in line`.